release-versions/generate_release_file.py

```diff
@@ -16,6 +16,8 @@ def url(repo, suffix):
 
 
 def get_versions(repository_name):
+    print(f"Fetching release branches for {repository_name}")
```

Comment:
I added this to do the initial debugging, but it's actually quite useful in general. I wouldn't add print statements to app code, but it seems quite handy to have it in a script. What do you think?

Reply:
Yeah, I think it makes sense to leave it here.